content/influxdb/v1.8/about_the_project/releasenotes-changelog.md

```diff
@@ -9,6 +9,21 @@ menu:
 v2: /influxdb/v2.0/reference/release-notes/influxdb/
 ---
 
+## v1.8.3 [2020-09-30]
+
+### Features
+
+- Use latest version of InfluxQL package.
+- Add `-lponly` flag to [`influx export`](/influxdb/v2.0/reference/cli/influx/export/) sub-command.
+- Add the ability to track number of values written via the [/debug/vars HTTP endpoint](/influxdb/v1.8/tools/api/#debug-vars-http-endpoint).
+- Update UUID library from [github.com/satori/go.uuid](https://github.com/satori/go.uuid) to [github.com/gofrs/uuid](https://github.com/gofrs/uuid).
+
+### Bug fixes
+
+- ArrayFilterCursor truncation for multi-block data.
+- Multi-measurement queries now return all applicable series.
+- Lock map before writes.
```

Comment:
Did we want to add the fixes in "quiet static analysis tools" issue--or did we leave out intentionally?:
- Remove redundant type in slice/array declarations.
- Call t.Fatal() from test-functions, not non-test go-routines.
- Remove unnecessary empty value operator from ranges.
- Call defer .Close() methods only after checking for error on Open().